openshift/templates/deploy.dc.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ${NAME}
+  annotations:
+    openshift.io/display-name: "wps-api"
+    description: "Wildfire Predictive Services - APIent"
+    openshift.io/long-description: "Wildfire Predictive Services - Web Client"
+    tags: "wps,wps-api"
+    iconClass: icon-js
+    openshift.io/provider-display-name: "Government of British Columbia"
+    openshift.io/documentation-url: "https://github.com/bcgov/wps-api"
+    openshift.io/support-url: "https://github.com/bcgov/wps-api"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}-${SUFFIX}
+parameters:
+  - name: NAME
+    description: Module name
+    value: wps-api
+  - name: SUFFIX
+    description: Deployment suffix, e.g. pr-###
+    required: true
+  - name: DOCKER_REGISTRY
+    value: docker-registry.default.svc:5000
+  - name: PROJECT_TOOLS
+    value: auzhsi-tools
+  - name: CPU_REQUEST
+    description: Requested CPU
+    value: 1000m
+  - name: CPU_LIMIT
+    description: CPU upper limit
+    value: 1000m
+  - name: MEMORY_REQUEST
+    description: Requested memory
+    value: 2Gi
```

Comment:
Same with memory - we're using 90mb when we're idling - and I bet it won't go much above that. 1Gi for now?